build_scripts/consul_setup/consul_setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Install Consul binaries
+cd /tmp/
+wget https://releases.hashicorp.com/consul/1.0.0/consul_1.0.0_linux_amd64.zip
+unzip consul_1.0.0_linux_amd64.zip
+sudo mv consul /usr/bin/
```

Comment:
Can we also do this in `/usr/local/bin`? System packages are normally installed in `/usr/bin`, user packages in `/usr/local/bin`